packs/familiar-abilities/resistance.json

```diff
@@ -18,7 +18,128 @@
             "remaster": true,
             "title": "Pathfinder Player Core"
         },
-        "rules": [],
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.TraitAcid",
+                        "value": "acid"
+                    },
+                    {
+                        "label": "PF2E.TraitCold",
+                        "value": "cold"
+                    },
+                    {
+                        "label": "PF2E.TraitElectricity",
+                        "value": "electricity"
+                    },
+                    {
+                        "label": "PF2E.TraitFire",
+                        "value": "fire"
+                    },
+                    {
+                        "label": "PF2E.TraitPoison",
+                        "value": "poison"
+                    },
+                    {
+                        "label": "PF2E.TraitSonic",
+                        "value": "sonic"
+                    }
+                ],
+                "flag": "resistanceOne",
+                "key": "ChoiceSet",
+                "priority": 50,
```

Comment:
```suggestion
```
We shouldn't need to worry about setting priorities for anything in this.

Reply:
I know you've already merged, but I'm curious if you don't mind me asking:

I thought priority in this case would help ensure that the `resistanceOne` ChoiceSet is always offered first before the `resistanceTwo`, since I only added the selection filtering to two to reduce the clutter.

I guess REs are just resolved in their declared order though... 😅

So priority is primarily used for the math operations in AE-Likes as you've mentioned in the wiki?